src/interface/api/Input.cpp

```diff
@@ -44,14 +49,24 @@ const std::vector<Drawing> Input::getDrawings() {
 }
 
 void Input::clearDrawings() {
-    std::lock_guard<std::mutex> lock(drawingMutex);
+    std::lock_guard<std::mutex> drawingLock(drawingMutex);
     drawings = {};
 }
 
 Input::~Input() {
     clearDrawings();
 }
 
+int Input::getFps() {
```

Comment:
This function name seems very contradictory... not sure what the class Input does exactly at the moment but this is definitely an output ;)